### ISSUE TITLE
can not convert and save videos

### ISSUE THREAD

**Issue by KasraCode** [shaked6540/YoutubePlaylistDownloader#206]
in 1.9.21 version it download all playlist and save them into <temp> folder (audio and video) but doesn't convert it into one file and save in destination

at the end thank you for making this usefull app ❤

**Comment by shaked6540** [comment]
Can you screenshot your download options? Also, you can try running the program as administrator, it usually solves most problems

**Comment by KasraCode** [comment]
yes that made it but when i tried to close the program, it erased the whole videos from temp folder and now i have to download them again

please fix this issue

**Comment by shaked6540** [comment]
I can't fix it if you dont give me all the information. Saying that a bug exist is not helping if it doesn't happen to me. Show me what settings you used to download, if you see an error message, what playlist you're trying to download etc..

**Comment by Maxi-Live** [comment]
Thanks for you awesome tool Shaked6540 :)

I just wanted to add I had the same issue but like you said running as Administrator fixed it for me. The conversion took place then.
